pkg/osquery/tables/wifi_networks/wifi_networks.go

```diff
@@ -0,0 +1,149 @@
+package wifi_networks
+
+import (
+	"bufio"
+	"bytes"
+	"context"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"github.com/go-kit/kit/log"
+	"github.com/go-kit/kit/log/level"
+	"github.com/kolide/launcher/pkg/dataflatten"
+	"github.com/kolide/launcher/pkg/osquery/tables/dataflattentable"
+	"github.com/kolide/launcher/pkg/osquery/tables/tablehelpers"
+	"github.com/kolide/launcher/pkg/osquery/tables/wifi_networks/internal"
+	"github.com/kolide/osquery-go"
+	"github.com/kolide/osquery-go/plugin/table"
+	"github.com/pkg/errors"
+)
+
+//go:generate go-bindata -nometadata -nocompress -pkg internal -o internal/assets.go internal/assets/
+
+type execer func(ctx context.Context, buf *bytes.Buffer) error
+
+type WlanTable struct {
+	client    *osquery.ExtensionManagerClient
+	logger    log.Logger
+	tableName string
+	getBytes  execer
+}
+
+func TablePlugin(client *osquery.ExtensionManagerClient, logger log.Logger) *table.Plugin {
+	columns := dataflattentable.Columns(
+		table.TextColumn("ssid"),
+	)
+
+	t := &WlanTable{
+		client:    client,
+		logger:    logger,
+		tableName: "kolide_wifi_networks",
+		getBytes:  execPwsh(logger),
+	}
+
+	return table.NewPlugin(t.tableName, columns, t.generate)
+}
+
+func (t *WlanTable) generate(ctx context.Context, queryContext table.QueryContext) ([]map[string]string, error) {
+	ctx, cancel := context.WithTimeout(context.Background(), 30*time.Second)
+	defer cancel()
+	var results []map[string]string
+	var output bytes.Buffer
+
+	err := t.getBytes(ctx, &output)
+	if err != nil {
+		return results, err
+	}
+	scanner := bufio.NewScanner(&output)
+	scanner.Split(tablehelpers.StanzaSplitter)
+	for scanner.Scan() {
+		chunk := scanner.Bytes()
+		rows, err := dataflatten.Ini(chunk, dataflatten.WithLogger(t.logger))
```

Comment:
It's in ini format!? Amazing!

Reply:
Ha I know, I'm not sure if it's 'strict ini' (probably no such thing), but it seems to be all simple key-value pairs, and the output isn't nested. 

The output is just the default that powershell's `Select-Object` prints out, and the objects aren't nested, so it seems to work.